examples/lvgldemo/Kconfig

```diff
@@ -11,6 +11,13 @@ menuconfig EXAMPLES_LVGLDEMO
 
 if EXAMPLES_LVGLDEMO
 
+config EXAMPLES_LVGLDEMO_CALIBRATE
+	bool "Calibrate touchscreen"
+	default y
```

Comment:
Change to n? more and more touch technology don't need calibrate anymore

Reply:
But most boards currently supported need it， like stm32f429i-disco, etc
If change it to n, we need to update these boards defconfig

Reply:
Ok.

Reply:
Yes, some boards that support the lvgl-demo are resistive touchscreen...